contracts/src/StakeTable.sol

```diff
@@ -21,6 +21,9 @@ contract StakeTable is AbstractStakeTable {
     /// account.
     error NodeAlreadyRegistered();
 
+    /// Error raised when a user tries to withdraw funds from a node that is not registered.
+    error NodeNotRegistered();
```

Comment:
Why are we changing this contract?. We have new `PermissionedStakeStable` contract for stake table

Reply:
The branches are just out of sync.